src/main/java/com/uber/jenkins/phabricator/conduit/Differential.java

```diff
@@ -119,7 +121,13 @@ public String getSummaryMessage(String phabricatorURL) {
 
     public String getPhabricatorLink(String phabricatorURL) {
         String revisionID = this.getRevisionID(true);
-        return String.format("%s%s", phabricatorURL, revisionID);
+        try {
+            URL base = new URL(phabricatorURL);
+            return new URL(base, revisionID).toString();
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+            return String.format("%s%s", phabricatorURL, revisionID);
```

Comment:
Yea, I see that now, the weird thing is that the string is formed both times by calling `this.getRevisionID(true)` which returns "DXXX". Could you perhaps open up the page source and make sure the HTML is being formed properly, and it's not some weird browser bug based on the fact that the slash is missing? Alternatively, try temporarily setting your Phabricator instance to have a trailing slash in your global settings and see if that fixes the problemS